tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/board/litex_vexriscv.overlay

```diff
@@ -0,0 +1,4 @@
+&eth0 {
```

Comment:
I see that we have this change in our project. Must be some git history quirk. Don't merge it.